apps/ui/src/components/Ui/WrapperInput.vue

```diff
@@ -1,18 +1,41 @@
 <script setup lang="ts">
-const props = defineProps<{
-  definition: any;
-  error?: string;
-  dirty?: boolean;
-}>();
+const props = withDefaults(
+  defineProps<{
+    definition: any;
+    inputLength?: number;
```

Comment:
To me `inputLength` is a bit weird name here, because `input` could mean two things (input element itself, or input's value - here we have the latter, but at least to me second definition is not intuitive) - it could just be called `valueLength` or `inputValueLength` and it should be clearer.